src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -717,21 +658,31 @@ protected internal override Expression VisitDefault(DefaultExpression node)
         {
             Out("default(");
             Out(node.Type.Name);
-            Out(")");
+            Out(')');
             return node;
         }
 
         protected internal override Expression VisitLabel(LabelExpression node)
         {
             Out("{ ... } ");
             DumpLabel(node.Target);
-            Out(":");
+            Out(':');
             return node;
         }
 
         protected internal override Expression VisitGoto(GotoExpression node)
         {
-            Out(node.Kind.ToString().ToLower());
```

Comment:
A lot of hidden allocs here: `Enum.GetName` which allocates a `string[]` for all the names to index into, and a conversion to lower case on top of it.